README.md

```diff
@@ -184,6 +185,60 @@ const fileLastModifiedDate = preval.require('./get-last-modified-date', '../../s
 const fileLastModifiedDate = '2017-07-04'
 ```
 
+### preval file comment (`// @preval`)
+
+**Before**:
+
+```javascript
+// @preval
+const fileLastModifiedDate = require('./get-last-modified-date')
```

Comment:
Hmmm, isn't this supposed to have a module.exports somewhere? Otherwise it's not a very useful module